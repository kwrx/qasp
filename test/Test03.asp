%@exists
a | na.
b | nb.
%@forall
c | nc.
d | nd.
e | ne.
%@constraints
{c;d;e}.
:- c, not e.
:- d, not e.
:- not c, not d, e.