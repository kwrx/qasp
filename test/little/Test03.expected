QASP 1.1

INCOHERENT
